engine/src/main/java/org/destinationsol/game/sound/OggSoundManager.java

```diff
@@ -89,12 +134,31 @@ public void play(SolGame game, PlayableSound playableSound, @Nullable Vector2 po
             position = source.getPosition();
         }
 
-        // Calculate the volume multiplier for the sound
-        float globalVolumeMultiplier = game.getCmp().getOptions().sfxVolumeMultiplier;
-        if (globalVolumeMultiplier == 0) {
```

Comment:
Why did you leave out 
`        if (globalVolumeMultiplier == 0) {
            return;
        }`
from the getVolume method? It still might speed up the process when Global Sound volume is zero? Or is it so that when global volume multiplier for sounds is zero all incoming requests are ignored at a higher level?

Reply:
Conclusion for future reference: It might speed it up a little bit, but it would make nearly unnoticeable difference, and the method isn't even called often much. (Except for beacons/engines that spam it like a hell, but these should be handled elsewhere).